nilearn/image/image.py

```diff
@@ -880,6 +880,18 @@ def clean_img(imgs, sessions=None, detrend=True, standardize=True,
     from .image import new_img_like
 
     imgs_ = check_niimg_4d(imgs)
+
+    # Check if t_r is set, otherwise propose t_r from imgs header
+    if low_pass is not None or high_pass is not None:
```

Comment:
```clean_img``` calls ```signal.clean```. May be we can improve the error message setting in ```signal.clean``` ?

This made me comment because I tried to run the script and signal.clean reports error message simply as 
"ValueError: Repetition time (t_r) must be specified for filtering". Not consistent with clean_img.

Reply:
I rewrote the `signal.clean` message with https://github.com/nilearn/nilearn/pull/1830/commits/d864d82cee67f669986839b2c781d80ea8c2e8ca but I don't see how the error message in `signal.clean` could know about a `t_r` value from a header. What kind of error message do you have in mind?

Reply:
Good point.

>What kind of error message do you have in mind?

What we rewrote is good to me.